packages/override_api_endpoint/README.md

```diff
@@ -0,0 +1,25 @@
+# override_api_endpoint
+
+[![override_api_endpoint pub.dev badge][pub-badge]][pub-badge-link]
+[![][build-badge]][build-badge-link]
+[![][codecov-badge]][codecov-badge-link]
+
+Overrides and persists default api endpoint for test environment.
+
+## Usage
+
+```dart
+final apiEndpoint = await overrideApiEndpoint(
```

Comment:
Please provide more info on how to use it. What are those parameters?

Reply:
Is done

Reply:
Done